src/Array.ts

```diff
@@ -705,7 +705,13 @@ export const rotate = <A>(n: number, xs: Array<A>): Array<A> => {
 }
 
 /**
- * Test if a value is a member of an array
+ * Test if a value is a member of an array. Takes a `Stetoid<A>` as a single
```

Comment:
there's a small type here: `Stetoid<A>`

Reply:
Thanks @raveclassic. Looks like there's another typo in the example. I think we can drop the prefix `setoid.` and go for

```
@example
import { setoidString } from 'fp-ts/lib/Setoid'

member(setoidString)(['thing one', 'thing two', 'cat in the hat'], 'thing two') // true
```

which also shows the required import

Reply:
Great catch @raveclassic and good call @gcanti - that's much clearer to show the dependency.

Updated. 👍